addons/website_slides/static/src/js/snippets.animation.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import publicWidget from '@web/legacy/js/public/public_widget';
+
+publicWidget.registry.WebsiteAnimate.include({
+    /**
+     * @override
+     */
+    getScrollingElement() {
+        if ($('.o_wslide_fs_article_content').length) {
+            return $('.o_wslide_fs_article_content:eq(0)');
```

Comment:
1. Don't search twice for the same element, create a variable with the result.
2. Don't use jQuery in new code anymore (it has been removed for the backend in 18.0 and is being removed in the frontend, it's easier to just never use it anymore).